src/import-csv.js

```diff
@@ -0,0 +1,56 @@
+import fs from 'fs';
+import Promise from 'bluebird';
+import CouchBulkImporter from 'couchbulkimporter';
+import parse from 'csv-parse';
+import BulkBadger from 'bulkbadger';
+
+export function cli (file, url, ...csvOptions) {
+  if (!file) {
+    const msg = [
+      'Usage:',
+      '',
+      'nmo import-csv [file] [couchdb-url] [...<csv options> <pairs>]',
+    ].join('\n');
+    const err = new Error(msg);
+    err.type = 'EUSAGE';
+
+    throw err;
+  }
+
+  return importcsv.apply(importcsv, [file, url, createOpts(csvOptions)]);
```

Comment:
option parsing works a bit different:

you can access all passed commands from the cli or the nmo.conf file using:

```
nmo.config.get('delimiter')
```

cli commands have a higher priority than values from the config file.

would this work:

```js
const opts = {delimiter: nmo.config.get('delimiter'), columns: nmo.config.get('columns')};
return importcsv.apply(importcsv, [file, url, opts]);
```

and then in ./bin/cli.js:

```js
var parsed = nopt({
  'json': [Boolean],
  'force': [Boolean],
  'delimiter': [String],
  'columns': [String]
}, {'v': 'v'}, process.argv, 2);
```

you can then delete the custom option parsing `createOpts` and it has the advantage that power users can define

```
[nmo]
json=true
```
 in their `.nmorc` as default value, and override it in special cases using the cli argument.

(they can use nmo to alter the config, using `nmo config set nmo json true`)